content/docs/2.0/concepts/scaling-jobs.md

```diff
@@ -35,11 +36,100 @@ spec:
     template:
       # describes the [job template](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
   pollingInterval: 30  # Optional. Default: 30 seconds
-  cooldownPeriod:  300 # Optional. Default: 300 seconds
-  minReplicaCount: 0   # Optional. Default: 0
+  successfulJobsHistoryLimit: 5 # Optional. Default: 100. How many completed jobs should be kept.
+  failedJobsHistoryLimit: 5 # Optional. Default: 100. How many failed jobs should be kept.
   maxReplicaCount: 100 # Optional. Default: 100
   triggers:
   # {list of triggers to create jobs}
 ```
 
 You can find all supported triggers [here](../scalers).
+
+## Details
+
+```yaml
+  jobTargetRef:
+    parallelism: 1 # [max number of desired pods](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#controlling-parallelism)
```

Comment:
Let's not link too much to pods since we allow folks to scale anything
```suggestion
    parallelism: 1 # Max number of desired instances ([docs](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#controlling-parallelism))
```